### PR TITLE
React Capstone: Component

### PR DESCRIPTION
**Pull request by Favourezeugwa**
## In this Project, I Implemented the following;
- Home page: When the page loads, the web app shows the list of categories that could be filtered out by the parameter that was input in the search bar, along with the category name, which will display some numeric values per category that comes from the API. The application navigates to the details page when the user clicks (or taps) on a category item.

- Details page: In the details page, the web app retrieves data from the API to show detailed data in the category.
When the user clicks on the "Back" button (<), the user navigates to the home page.

- Created unit tests for pure functions in your Redux code.
- Create integration tests for your application using the React Testing Library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,74 @@
...
-### `npm start`
+### Screenshots
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[link to live demo on Netlify]
```
Comment (DammyShittu) [inline README.md]: - [ ] You are expected to deploy your app and add the link here. Kindly do that.

File: src/Components/Details/Details.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  .detailTitle {
+    padding: 5px 10px;
+    font-size: 20px;
+  }
+
+  .imageContainer {
+    padding: 30px;
+  }
+}
```
Comment (DammyShittu) [inline src/Components/Details/Details.css]: - [ ] According to the expected design, kindly add space between the attributes and their values.

  Your Implementation | Provided Design
  :---:|:---:
  details-1 | details-2

File: README.md [hunk 3]
```diff
@@ -1,70 +1,74 @@
...
 
-### Analyzing the Bundle Size
+Give a ⭐️ if you like this project!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+## Acknowledgments
 
-### Making a Progressive Web App
+- Inspiration(Microverse Math magicians Wireframe)
+- Microverse student community
```
Comment (DammyShittu) [inline README.md]: - [ ] Kindly include an acknowledgment for Nelson Sakwa, the design author,

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Favourezeugwa,

## Highlights
✔️ Search functionality
✔️ Descriptive PR

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (Favourezeugwa) [reply README.md]: Thank you @DammyShittu!

File: README.md [hunk 4]
```diff
@@ -1,70 +1,78 @@
...
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+![](./src/Assets/MobileHome.snip.PNG)
+![](./src/Assets/MobileDetails2.snip.PNG)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Demo
 
-### `npm test`
+[link to live demo on Netlify](https://crypto-metrics-webapp.netlify.app/)
```
Comment (smunozmo) [inline README.md:16]: [OPTIONAL]] Please make sure you update your deployment after merging the branch as you said, this is a very nice project to display in your portfolio, you did a great job! 😎

**Review by smunozmo — APPROVED** [review]
Hi @Favourezeugwa, 👋 

## Highlights 👏 

- Nice job finishing your project, everything is looking good and working properly, congrats! 🎉 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

# APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 4 above)
Comment (Favourezeugwa) [reply README.md:16]: Thank you @smunozmo 😃, Will add the life demo link for sure.

**Pull request merged**